release/release_test.go

```diff
@@ -177,41 +170,56 @@ func (s *ReleaseTestSuite) TestReleaseInfo(c *C) {
 }
 
 func (s *ReleaseTestSuite) TestFilesystemRootType(c *C) {
-	reported_type, err := release.FilesystemRootType()
+	output, err := exec.Command("df", "--output=fstype", "/").CombinedOutput()
 	c.Assert(err, IsNil)
+	// $ df --output=fstype / # Example output on WSL1
+	// Type
+	// wslfs
+
+	fields := strings.Fields(string(output))
+	c.Check(len(fields), Equals, 2)
 
-	// From man stat:
-	// %t   major device type in hex, for character/block device special files
-	output, err := exec.Command("stat", "-f", "-c", "%t", "/").CombinedOutput()
+	wants := strings.TrimSpace(fields[1])
 	c.Assert(err, IsNil)
 
-	outstr := strings.TrimSpace(string(output[:]))
-	statted_type, err := strconv.ParseInt(outstr, 16, 64)
+	got, err := release.FilesystemRootType()
 	c.Assert(err, IsNil)
 
-	c.Check(reported_type, Equals, statted_type)
+	c.Check(wants, Equals, got)
 }
 
 func (s *ReleaseTestSuite) TestNonWSL(c *C) {
-	defer mockWSLsetup(c, false, false, ext4)()
```

Comment:
Sorry, again pre-existing tests so not really your fault :) I would use something like:
```
        existsWSLinterop := false
        existsRunWSL := false
	defer mockWSLsetup(c, existsWSLinterop, existsRunWSL, "ext4")()
```
here for all the touched tests to make it clear what the bools are about. But maybe a followup because it already exists.